### PR TITLE
Get CentOS8 kickstart to work out-of-box

### PR DESCRIPTION
**Pull request by lzap**
There is a long discussion at

https://community.theforeman.org/t/satellite-cannot-provisioning-centos-8-server/18091

but in short: CentOS8 provisioning does not work out of box and I believe this must work without any manual steps. This is what Katello recommends in its guide:

https://theforeman.org/plugins/katello/3.14/user_guide/provisioning/index.html

I made it a little bit safer by expecting that OS name is "CentOS" (any case) not to break RHEL provisioning in any way.

### CODE REVIEW

**Review by ianballou — COMMENTED** [review]
This worked fine for me! I only tested the rendering of the template, but as long as the `repo` line for AppStream is there it should install just fine.

Just had one question about the formatting.

File: provisioning_templates/provision/kickstart_default.erb [hunk 1]
```diff
@@ -75,8 +75,13 @@ liveimg --url=<%= liveimg_url %> <%= proxy_string %>
 <% else %>
 <%= @mediapath %><%= proxy_string %>
 <% @additional_media.each do |medium| -%>
+  <% if rhel_compatible && @host.operatingsystem.name.downcase.include?("centos") && os_major >= 8 && medium[:url] && medium[:url].include?("AppStream") -%>
```
Comment (ianballou) [inline provisioning_templates/provision/kickstart_default.erb]: Is it normal to have the indentation like this?

Comment (mmoll) [reply provisioning_templates/provision/kickstart_default.erb]: I de-indented it.

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**